Upstream-Status: Pending
--- a/fb_ili9488.c
+++ b/fb_ili9488.c
@@ -24,7 +24,7 @@
 #include <linux/module.h>
 #include <linux/kernel.h>
 #include <linux/init.h>
-
+#include <linux/gpio/consumer.h>
 #include <linux/gpio.h>

 #include "fbtft.h"
@@ -105,8 +105,8 @@ static int write_vmem16_18bus8(struct fbtft_par *par, size_t offset, size_t len)
 	remain = len / 2;
 	vmem16 = (u16 *)(par->info->screen_buffer + offset);

-	if (par->gpio.dc != -1)
-		gpio_set_value(par->gpio.dc, 1);
+	if (par->gpio.dc)
+		gpiod_set_value(par->gpio.dc, 1);

 	/* number of pixels that fits in the transmit buffer */
 	tx_array_size = par->txbuf.len / 3;
